src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -411,19 +404,26 @@ def _parse_function_args(cls, func):
 
     @classmethod
     def endpoints(cls):
+        """
+        Endpoints are functions or methods which have been decorated with the
+        @cherrpy.expose decorator and hence have the `exposed` attribute set to
```

Comment:
This is not always true. This is only true for the implementation of this method in this class. Sub classes might have a different implementation. Therefore, the comment should be more generic instead of referring to an implementation detail such as `@cherrypy.expose`